freesound/settings.py

```diff
@@ -334,14 +339,65 @@
                 'freesound.context_processor.context_extra',
             ],
         },
+        'NAME': FRONTEND_NIGHTINGALE,
+    },
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [
+            os.path.join(os.path.dirname(__file__), '../templates2'),
+        ],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.contrib.auth.context_processors.auth',
+                'django.template.context_processors.debug',
+                'django.template.context_processors.i18n',
+                'django.template.context_processors.media',
+                'django.template.context_processors.static',
+                'django.template.context_processors.tz',
+                'django.template.context_processors.request',
+                'django.contrib.messages.context_processors.messages',
+                'freesound.context_processor.context_extra',
+            ],
+        },
+        'NAME': FRONTEND_BEASTWHOOSH,
     },
+
 ]
 
+
+MESSAGE_STORAGE = 'django.contrib.messages.storage.cookie.CookieStorage'
+
+# We use the last restart date as a timestamp of the last time freesound web was restarted (lat time
+# settings were loaded). We add this variable to the context processor and use it in base.html as a
+# parameter for the url of all.css and freesound.js files, so me make sure client browsers update these
+# files when we do a deploy (the url changes)
+LAST_RESTART_DATE = datetime.datetime.now().strftime("%d%m")
+
+# Followers notifications
+MAX_EMAILS_PER_COMMAND_RUN = 1000
+NOTIFICATION_TIMEDELTA_PERIOD = datetime.timedelta(days=7)
+
+PASSWORD_HASHERS = [
```

Comment:
Why are there things in here merged from the main branch. Shouldn't we be rebasing so these things don't show up in the PRs?

Reply:
I think because I moved things arround in settings and then merged master back to the branch.